test-cases/longevity/longevity-profile-d-multidc.yaml

```diff
@@ -0,0 +1,35 @@
+test_duration: 10000
+
+service_level_shares: [500, 500, 900]
+
+prepare_write_cmd:  ["cassandra-stress user profile=scylla-qa-internal/profile-d/profile-top.yaml no-warmup 'ops(insert=1)' n=35104000 -mode cql3 native <sla credentials 0> -rate threads=96"]
+
+stress_cmd: ["cassandra-stress user profile=scylla-qa-internal/profile-d/profile-cache.yaml     'ops(readq=1)'  cl=LOCAL_QUORUM no-warmup duration=4h -mode cql3 native <sla credentials 1> -rate fixed=615/s threads=32   ",
+             "cassandra-stress user profile=scylla-qa-internal/profile-d/profile-cold-read.yaml 'ops(readq=1)'  cl=LOCAL_QUORUM no-warmup duration=4h -mode cql3 native <sla credentials 0> -rate fixed=7329/s threads=96  ",
+             "cassandra-stress user profile=scylla-qa-internal/profile-d/profile-write.yaml     'ops(insert=1)' cl=LOCAL_QUORUM no-warmup duration=4h -mode cql3 native <sla credentials 3> -rate fixed=41880/s threads=32 ",
+             ]
+
+n_db_nodes: '3 3 3 3 3'
+n_loaders: '1 1 1 1 1'
+region_aware_loader: true
+n_monitor_nodes: 1
+availability_zone: 'a,b,c'
+
+
+
+#round_robin: true
```

Comment:
Why this line is here? Current set of stress commands is not compatible with the `round_robin=true`.

Reply:
removed